### PR TITLE
Error Fix

### PR DESCRIPTION
**Pull request by Wilamaxin**
I fixed some errors

### CODE REVIEW

File: features/compact-editor.js [hunk 1]
```diff
@@ -11,17 +11,18 @@
 
 var style = document.createElement("style");
 style.innerHTML = `
-.gui_target-wrapper_36Gbz {
-  padding-top: 0.25rem !important;
-}
```
Comment (rgantzos) [inline features/compact-editor.js:16]: Why did you remove this?

**Review by rgantzos — CHANGES_REQUESTED** [review]

File: features/compact-editor.js (hunk 1 above)
Comment (Wilamaxin) [reply features/compact-editor.js:16]: It looks bad and doesn't necessarily make the editor more compact

**Comment by Wilamaxin** [comment]
Results of changes in "Compact Editor" and "Sprite Clones":

- Before (Only Compact Editor)
image

- After (Only Compact Editor)
image

- Before (Compact Editor +  Sprite Clones)
image

- After (Compact Editor +  Sprite Clones)
image

**Review by rgantzos — APPROVED** [review]

File: features/compact-editor.js (hunk 1 above)
Comment (Wilamaxin) [reply features/compact-editor.js:16]: Notice the space between the stage and the properties


Before

image

After

image

**Review by rgantzos — CHANGES_REQUESTED** [review]
The compact editor changes aren't working very well for me.
Screenshot 2023-04-23 at 11 42 41 AM

File: features/sprite-clones.js [hunk 2]
```diff
@@ -9,21 +9,21 @@ if (
...
       ).textContent = `${(
-        ScratchTools.Scratch.vm.editingTarget.sprite.clones.length - 1
+        ScratchTools.Scratch.vm.editingTarget?.sprite.clones.length - 1
       ).toString()} clones`;
     } else {
       var foundIt = false;
       document.querySelectorAll("div").forEach(function (el) {
         if (foundIt === false) {
-          if (el.className.startsWith("sprite-info_row_")) {
+          if (el.className.startsWith("sprite-info_row_1om5V") &&! el.className.includes("sprite-info_row-primary_10JrS")) {
```
Comment (rgantzos) [inline features/sprite-clones.js:18]: Why did you change the selector for this one?

**Comment by Wilamaxin** [comment]
It looks exactly like the unchanged one on that screenshot. Make sure that you tested it with both files updated

**Comment by rgantzos** [comment]
> It looks exactly like the unchanged one on that screenshot. Make sure that you tested it with both files updated

I don't think you pushed all the changes you meant to. The commit that claims to make them both compatible only changes some selectors, nothing that alters the features. The screenshot I shared is based on your fork.

File: features/sprite-clones.js (hunk 2 above)
Comment (Wilamaxin) [reply features/sprite-clones.js:18]: It's divided like that:
image
image
And the old script was inserting the clone counter after the first box, I changed it to insert the button after the second box, so that it wouldn't break in half

**Comment by Wilamaxin** [comment]
It works as intended for me. I checked that on 3 different computers.

**Comment by Wilamaxin** [comment]
I tested it again and I'm sure that it's a problem on your side

**Comment by Daniel4-Scratch** [comment]
image
works fine for me
